### PR TITLE
Update configure-remediation-microsoft-defender-antivirus.md

### LINKED ISSUE #12179: Contradicting information regarding Quarantine data retention timeframe in Microsoft defender
This is the exact text from Group policy setting in Windows 11 22631.1830

>  If you disable or do not configure this setting, items will be kept in the quarantine folder indefinitely and will not be automatically removed.

image

But [this web page](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/configure-remediation-microsoft-defender-antivirus) says Quarantined items will be deleted after 90 days by default.

image

I'm not using Defender for Endpoint, I use the normal Microsoft Defender for consumer, and the page says it also applies to Microsoft Defender.

image

So I'm reporting this contradicting info to be reviewed and fixed, thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 77c974b4-7004-d52c-0ffe-c6f3729477f9
* Version Independent ID: 7162a35d-fc78-54ac-531a-02ec8c07a8ff
* Content: [Configure remediation for Microsoft Defender Antivirus detections](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/configure-remediation-microsoft-defender-antivirus?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/configure-remediation-microsoft-defender-antivirus.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/configure-remediation-microsoft-defender-antivirus.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the default or not configured info for quanrentine

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12179

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-remediation-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ You can also use the [`Set-MpPreference` PowerShell cmdlet](/powershell/module/d
 |Scan|Create a system restore point|A system restore point will be created each day before cleaning or scanning is attempted|Disabled|
 |Scan|Turn on removal of items from scan history folder|Specify how many days items should be kept in the scan history|30 days|
 |Root|Turn off routine remediation|You can specify whether Microsoft Defender Antivirus automatically remediates threats, or if it should ask the endpoint user what to do.|Disabled (threats are remediated automatically)|
-|Quarantine|Configure removal of items from Quarantine folder|Specify how many days items should be kept in quarantine before being removed|90 days|
+|Quarantine|Configure removal of items from Quarantine folder|Specify how many days items should be kept in quarantine before being removed|Items will be kept in the quarantine folder indefinitely and will not be automatically removed.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-remediation-microsoft-defender-antivirus.md]:
```suggestion
|Quarantine|Configure removal of items from Quarantine folder|Specify how many days items should be kept in quarantine before being removed|Items will be kept in the quarantine folder indefinitely and will not be automatically removed|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**